src/status_im/ui/screens/chat/stickers/views.cljs

```diff
@@ -19,10 +20,10 @@
   [react/view {:style {:flex 1 :align-items :center :justify-content :center}}
    [vector-icons/icon :icons/stickers-big {:color colors/gray}]
    [react/text {:style {:margin-top 8 :font-size 17}} (i18n/label :t/you-dont-have-stickers)]
-   [react/touchable-highlight {:on-press #(do
-                                            (re-frame/dispatch [:stickers/load-packs])
-                                            (re-frame/dispatch [:navigate-to :stickers]))}
-    [react/text {:style {:margin-top 17 :font-size 15 :color colors/blue}}
+   [react/touchable-opacity {:on-press #(do
+                                          (re-frame/dispatch [:stickers/load-packs])
+                                          (re-frame/dispatch [:navigate-to :stickers]))}
+    [react/text {:style {:margin-top 17 :font-size 15 :color colors/blue :height 20}}
```

Comment:
hey, just wondering is `height 20` the entire touchable area of the button, if so that's too small, should be a 44, below that we risk of people missing the button with their fingers :) you can then reduce the top margin to 6, thx!

Reply:
good catch, thanks

Reply:
@errorists When do you submit your first code PR?? :D